test/integration/data/fsync.js

```diff
@@ -0,0 +1,72 @@
+const tester = require('../../__fixtures/utils/test_helper').create();
```

Comment:
These tests are not adequate yet, we need to use a spy to check that fsync is indeed called based on these parameters, also we need to check fsync works for configurations with multiple datastores, ie: https://github.com/happner/happn-3/blob/master/test/integration/data/multiple_datasource.js